great_expectations_cloud/agent/agent.py

```diff
@@ -113,7 +114,7 @@ class GXAgent:
     _PYPI_GX_AGENT_PACKAGE_NAME = "great_expectations_cloud"
     _PYPI_GREAT_EXPECTATIONS_PACKAGE_NAME = "great_expectations"
 
-    def __init__(self: Self):
+    def __init__(self: Self, enable_progress_bars: bool = True) -> None:
```

Comment:
What's the mechanism to actually disable this?
A CLI argument?
Or is the plan for the GX-Runner to disable this by default?

Reply:
The runner overrides `__init__()` so would need to be handled differently there.